docs/networking/native-ipv6-networking.md

```diff
@@ -92,27 +95,48 @@ While default IPv6 addresses are configured automatically, you will need to stat
       auto lo
       iface lo inet loopback
 
-      # The primary network interface (IPv4)
+      # Automatically brings up the default network interface (eth0)
       auto eth0
-      iface eth0 inet dhcp
```

Comment:
I thought it would add confusion to include the IPv4 configuration when this guide is focused on v6. The lines I left in are just for the interface, which IPv6 would need to work. In retrospect, this could add more confusion if some readers removed this section, disabling IPv4. 

Reverting my changes to what the guide originally said. I tested it to make sure it works on both distros.